src/main/java/org/team1540/robot2023/commands/arm/SetRotation.java

```diff
@@ -0,0 +1,39 @@
+package org.team1540.robot2023.commands.arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.team1540.robot2023.utils.RollingAverage;
+
+public class SetRotation extends CommandBase {
+    private final double threshold = 10;
+    private final RollingAverage rollingAverage = new RollingAverage(10);
+
+    private final Arm arm;
+
+    private final double angle;
+
+    public SetRotation(Arm arm, double x, double y) {
```

Comment:
Here, it might be more useful to either make the arguments the angle of the arm, or rename the method to something like "setArmAlignmentVector" to make it more intuitive to use